style.css

```diff
@@ -113,766 +179,616 @@ a {
+  0% {
+    transform: rotate(0);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
 }
 
-.div-text {
+.body {
   display: flex;
-  position: absolute;
   flex-direction: column;
-  margin-top: 20px;
-  text-align: center;
-  justify-content: center;
-  font-family: Poppins, sans-serif;
-  font-weight: normal;
-  width: 380px;
-  padding-top: 30px;
+  height: 610px;
+  background-image: url("./image/header_background.svg");
+  background-repeat: no-repeat;
 }
 
-.mob {
-  display: block;
+.bodyTrans {
+  background-image: url('./image/fade_back.svg');
+  background-color: #6070ff;
 }
 
-.dec {
-  display: none;
+.name {
+  margin: 0 24px 12px 24px;
+  line-height: 52px;
+  color: #172b4d;
 }
 
-.br {
-  display: block;
+.name h1 {
+  margin: 90px 0 0 0;
+  font-size: 40px;
 }
 
-.internal-link-dec {
-  display: none;
+.name h2 {
+  margin: 0 0 0 0;
+  font-size: 40px;
 }
 
-.nav-dec {
-  display: none;
-}
+.description {
+  font-family: Popinns-light, Arial, Helvetica, sans-serif;
+  margin: 0 24px 0 24px;
+  margin-top: 0;
 
-.title-home-page {
-  font-family: Poppins, sans-serif;
-  font-weight: bold;
-  font-size: 50px;
-  color: rgb(30, 29, 79);
-  line-height: 60px;
-  text-align: start;
-  justify-content: center;
-  margin-top: 40px;
-  margin-left: 20px;
-  margin-right: 20px;
+  /* width: 327px; */
+  height: 150px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
 }
 
-.dec-home-page {
-  font-family: Poppins, sans-serif;
-  font-weight: normal;
-  font-size: 17px;
-  line-height: 24px;
-  color: rgb(30, 29, 79);
-  margin-top: -20px;
-  text-align: start;
-  margin-left: 20px;
-  margin-right: 20px;
+.contact {
+  margin: 12px 0 12px 24px;
+  color: #142cff;
 }
 
-.footer h3 {
-  font-family: Poppins, sans-serif;
-  font-weight: normal;
-  font-size: 16px;
-  color: rgb(127, 140, 255);
-  text-align: start;
-  justify-content: center;
-  margin-left: 20px;
-  margin-right: 20px;
+.connect-logo {
+  margin-left: 24px;
+  display: flex;
 }
 
-.contact ul {
+.logo-div {
   display: flex;
-  align-items: center;
+  justify-content: space-between;
+  width: 163.5px;
+  list-style-type: none;
   padding-left: 0;
-  color: aqua;
-  margin-right: 20px;
-  margin-left: 10px;
+  margin-top: 0;
 }
 
-.icon {
-  list-style-type: none;
-  padding: 10px;
+/* WORK SECTION */
+.work_section {
+  display: grid;
+  grid-template-columns: auto;
+
+  /* grid-template-columns: auto; */
+
+  /* flex-direction: column; */
+  justify-content: space-between;
+  margin-top: 26px;
 }
 
-.works {
-  display: flex;
+.project {
+  /* padding-top: 80px; */
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.project1 {
+  /* display: flex; */
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  margin: 30px;
-  row-gap: 30px;
+  align-items: flex-end;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-top: 16px;
+  width: 100%;
+  margin: 0 0 0 0;
 }
```

Comment:
Great job team 🤝

- [ ] Kindly add a margin at the bottom of your mobile popup design so as to show all the elements on the page.
- [ ] Kindly make sure the font family and color are similar or close to what is provided in the Figma template.
- Your design
![mobile popup](https://user-images.githubusercontent.com/63917725/181723895-b680eb0f-55e8-49a5-82b0-bae555b3e92e.PNG)

- Figma design
![Figma mobile](https://user-images.githubusercontent.com/63917725/181724088-98b39e94-afec-49ef-888e-9e6e81983bc8.PNG)